addons/api/tests/unit/models/worker-test.js

```diff
@@ -71,6 +71,38 @@ module('Unit | Model | worker', function (hooks) {
     const model = store.peekRecord('worker', workerId);
     await model.setApiTags(tags);
   });
+  
+  test('it has a `removeApiTags` method that targets a specific POST API', async function (assert) {
+    assert.expect(1);
+    const workerId = 'w_123';
+    const tags = {
+      tag1: ['value1'],
+    };
+    this.server.post(
```

Comment:
What is the purpose of mocking a `post` request within this test?

Reply:
We are mocking the `post` so we can assert that the correct body is being added to the payload when we call `/workers/WORKER_ID:remove-worker-tags`.

Reply:
I answered this one on the comment above, my bad mixxing this 2 comments but they are related.